api/app/package.json

```diff
@@ -26,7 +26,7 @@
     "@aws-sdk/client-cloudwatch": "^3.338.0",
     "@medplum/core": "^2.0.18",
     "@medplum/fhirtypes": "^2.0.18",
-    "@metriport/api": "^3.1.6",
+    "@metriport/api": "^4.0.2-alpha.0",
```

Comment:
do we need to update this to be the proper release version?

Reply:
Oh yea we will need to deploy the proper one i can do that